### ISSUE TITLE
There is no way to access what vscrt is finally used in the build

### ISSUE THREAD

**Issue by viric** [mesonbuild/meson#8493]
**Describe the bug**
b_vscrt allows values dependant on buildtype (from_buildtype, static_from_buildtype, ...) but it is not accessible what crt it ends up using.

Moreover, cc.get_define('_DEBUG') will not work either because that function forgets the CRT flags in the test.

**To Reproduce**
-

**Expected behavior**
There should be a way (maybe cc.get_define) to learn what CRT has been finally chosen by meson, given the values in b_vscrt, without replicating in meson build files the internal python logic around b_vscrt & buildtype.

**system parameters**
* native windows build
* win 10
* python 3.9.2
* meson 0.57.1
* ninja 1.10.2

**Comment by bruchar1** [comment]
Not sure if something changed since meson 0.57.1, but I can tell that `get_option('b_vscrt')` gives you the actual value (md, mt, mdd, mtd, or none), and not `from_buildtype` or `static_from_buildtype`.

**Comment by xclaesse** [comment]
> I can tell that `get_option('b_vscrt')` gives you the actual value (md, mt, mdd, mtd, or none), and not `from_buildtype` or `static_from_buildtype`.

I don't think that's true:
```meson
message(get_option('b_vscrt'))
```

gives:
```
Message: from_buildtype
```

But I think we should actually do as you said.

**Comment by bruchar1** [comment]
@xclaesse You're right. I didn't realized the parameter was set in my machine file. Will put this in my todo list...
